resources.md

```diff
@@ -14,16 +14,15 @@ show_hero: true
 
 ### Resources
 
-
 * [Client libraries](#libraries)
 * [BI tools and notebooks](#ui)
 * [Management tools](#tools)
 * [Enterprise support](#support)
 * [Managed services](#managed)
 * [Cloud](#cloud)
-  * [AWS](#aws)
-  * [Azure](#azure)
-  * [GCP](#gcp)
+  * [Amazon Web Services](#aws)
+  * [Google Cloud](#gcp)
+  * [Microsoft Azure](#azure)
```

Comment:
This should be in a separate commit. It's unrelated to adding the Starburst entry